FAQ.md

```diff
@@ -82,19 +81,17 @@ backend:
         - amplifyPush --simple
 ```
 
-#### How to migrate domains to Amplify with minimal downtime
-
-[The best process to follow to minimize downtime here would be](#how-to-migrate-domains-to-amplify-with-minimal-downtime): 
+#### How do I migrate domains to Amplify with minimal downtime?
```

Comment:
Looks like we need to re-add the new anchor tag `#how-do-i-migrate-domains-to-amplify-with-minimal-downtime`

Reply:
That link ref was to the same section. Thought that it might have been a typo in the prev version

Reply:
Oh I see it was linking to itself